Gemfile

```diff
@@ -58,6 +59,5 @@ group :development, :test do
   gem 'spring'
   gem 'capybara'
   gem 'mocha'
-  gem 'pry'
   gem 'launchy'
 end
```

Comment:
So, I found this commentary on it: http://robots.thoughtbot.com/a-healthy-bundle
TL;DR: unless you are sure the gem has an excellent history of backwards compatibility (e.g. pg), it is a good idea to use "pessimistic versioning"--e.g. gem 'nokogiri', '~> 1.0.3'.